README.md

```diff
@@ -1 +1,118 @@
-# portfolio_v2
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>"Portfolio" project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio: setup and mobile first] <a name="Portfolio: setup and mobile first"></a>
+
+**[portfolio_v2]** This is my portfolio website project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML, CSS, GIT
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:alex1779/portfolio_v2.git
+  
+```
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
```

Comment:
- [ OPTIONAL ] Since you are the only contributor to this project, it would be better if you change `Authors` to `Author` to prevent typo in your readme file :smile: